packages/analytics-types/src/config.ts

```diff
@@ -77,16 +77,26 @@ export interface ReactNativeTrackingOptions extends TrackingOptions {
 
 export interface AdditionalBrowserOptions {
   attribution?: AttributionBrowserOptions;
+  trackPageViews?: boolean | PageTrackingBrowserOptions;
 }
 
 export interface AttributionBrowserOptions {
   disabled?: boolean;
   excludeReferrers?: string[];
   initialEmptyValue?: string;
+  resetSessionOnNewCampaign?: boolean;
+  /** @deprecated New campaigns are now always tracked by default. Setting this to true also sets resetSessionOnNewCampaign to true. */
   trackNewCampaigns?: boolean;
+  /** @deprecated Use the new top level trackPageViews configuration instead. */
   trackPageViews?: boolean;
 }
 
+export interface PageTrackingBrowserOptions {
+  filter?: PageTrackingFilter;
+}
+
+export type PageTrackingFilter = 'onAttribution' | (() => boolean) | undefined;
```

Comment:
i think it's better to do

```ts
export type PageTrackingFilter = 'onAttribution' | (() => boolean);
```

and leave it to the usage whether to allow undefined or not like

```ts
filter: PageTrackingFilter
```

or 

```ts
filter?: PageTrackingFilter
```

Reply:
Done